torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -677,7 +704,11 @@ def create_from_config(cls, config: CppMicroGemmConfig):
                 continue
             if (
                 config.input_dtype == input_dtype
-                and config.output_dtype == output_dtype
+                and (
+                    config.output_dtype == torch.int32
+                    if input_dtype == torch.uint8
+                    else config.output_dtype == output_dtype
+                )
```

Comment:
Is it better to check this from the caller side? The caller makes sure that the `output_dtype` is torch.int32 in the case of int8 gemm?

Reply:
Thanks for the comment. I think this line of change is not needed anymore. Remove this change in this PR.